package.json

```diff
@@ -1,10 +1,16 @@
 {
   "name": "reactnative_boilerplate",
-  "version": "0.1.0",
+  "version": "1.7.0",
```

Comment:
Why update the boilerplace version?

Reply:
Strange, probably a typo. Didn't mean to change it